client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks/use-categories-order.ts

```diff
@@ -0,0 +1,45 @@
+import { useMemo } from 'react';
+import type { Category } from '../types';
+
+const patternCategoriesOrder = [
```

Comment:
I copied the implementation from the editor because I find it "fine" https://github.com/WordPress/gutenberg/pull/47760/files

Reply:
Got it, thanks for the explanation!